GOVERNANCE.md

```diff
@@ -52,40 +48,14 @@ and so on.
 
 The current committers are (in addition to the TC members above):
 
-* [@hugs](https://github.com/hugs)
```

Comment:
taking the opportunity to clean up this list as well. feel free to suggest other modifications that i have missed.

Reply:
do we have a list of honourable mentions that we keep forever? I would prefer people that ever were part of Appium development to still be mentioned, especially these that contributed a lot.

Reply:
Yes, I also think it's better to have a section for former contributors/honorable mentions, instead of removing them

Reply:
yeah, great idea!

Reply:
ok, this is done now.